internal/syncer/connector/azure_devops.go

```
@@ -0,0 +1,1055 @@
+}
+
+func (c *AzureDevOpsConnector) indexesPullRequests() bool {
+	return c.contentTypes == azureDevOpsContentPullRequests || c.contentTypes == azureDevOpsContentBoth
+}
+
+func (c *AzureDevOpsConnector) checkSettings() error {
+	if c == nil {
+		return fmt.Errorf("azure devops connector is nil")
+	}
+	if c.organization == "" {
+		return fmt.Errorf("Invalid connector settings: 'organization' must be provided")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)